### ISSUE TITLE
PyTorch Mobile speed_benchmark_torch crashes on mobile optimized model

### ISSUE THREAD

**Issue by windmaple** [pytorch/pytorch#48410]
## To Reproduce

1. Build binary using:
BUILD_PYTORCH_MOBILE=1 ANDROID_ABI=arm64-v8a ./scripts/build_android.sh -DBUILD_BINARY=ON

2. Export MobileNet v2:
import torch
from torch.utils.mobile_optimizer import optimize_for_mobile
model = torch.hub.load(‘pytorch/vision:v0.6.0’, ‘mobilenet_v2’, pretrained=True)
model.eval()

scriptedm = torch.jit.script(model)
torch.jit.save(scriptedm, “mobilenet_v2_scripted.pt”)
optedm = optimize_for_mobile(scriptedm)
torch.jit.save(optedm, “mobilenet_v2_opted.pt”)

3. Run the optimized model:
(python3.7) user1$ adb shell “/data/local/tmp/speed_benchmark_torch --model=/data/local/tmp/mobilenet_v2_opted.pt” --input_dims=“1,3,224,224” --input_type=“float”
Starting benchmark.
Running warmup runs.
Main runs.
Segmentation fault

mobilenet_v2_scripted.pt however works fine. Device is Pixel 3a on Android 10.

## Expected behavior

Well, it shouldn't crash

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0): 4ed7f36ed in master
 - OS (e.g., Linux): Android 10
 - How you installed PyTorch (`conda`, `pip`, source): N/A
 - Build command you used (if compiling from source): BUILD_PYTORCH_MOBILE=1 ANDROID_ABI=arm64-v8a ./scripts/build_android.sh -DBUILD_BINARY=ON
 - Python version: 3.7
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
